[0.573s] Invoking command in '/home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1': PYTHONPATH=/home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1/prefix_override:/home/kurzmann/maze_bot/programming/mbot_ws/install/mazebot_v1/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py develop --uninstall --editable --build-directory /home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1/build
[0.584s] /usr/bin/python3: can't open file '/home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1/setup.py': [Errno 2] No such file or directory
[0.586s] Invoked command in '/home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1' returned '2': PYTHONPATH=/home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1/prefix_override:/home/kurzmann/maze_bot/programming/mbot_ws/install/mazebot_v1/lib/python3.10/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py develop --uninstall --editable --build-directory /home/kurzmann/maze_bot/programming/mbot_ws/build/mazebot_v1/build
